packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import {
+  Preview,
+  Canvas,
+  Props,
+  Story,
+  Meta,
+} from '@storybook/addon-docs/blocks';
+import { ArgsTable } from '@storybook/components';
+import { VtmnBadge } from '@vtmn/svelte';
+
+<Meta title="Components / Indicators / VtmnBadge / Documentation" />
+
+# Badge
+
+## Quick start
+
+You need to add `@vtmn/svelte` on your `package.json`
+
+```
+npm install @vtmn/svelte
+```
+
+Now, you can use `VtmnBadge` on your svelte file with
+
+```svelte
+<script>
+  import { VtmnBadge } from '@vtmn/svelte';
+</script>
+```
+
+## Value
+
+There are 3 different types of display
+
+- When value is undefined or 0
+- When value is between 1 and 99
+- When value is upper than 99
+
+<Canvas>
+  <Story name="No value">
```

Comment:
Why it is named "No value" ?

Reply:
Humm I can rename to "Equal to zero"

Reply:
With our discussion, the conversation are resolved ?